### DISCUSSION TITLE
Renovate assings CODEOWNERs even with automerge enabled

### DISCUSSION THREAD

**Discussion by mrclrchtr** [renovatebot/renovate#25378] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

As written in the headline. If automerge is on and the bot also recognizes that automerge should work, the CODEOWNERS are still assigned:

<img width="644" alt="image" src="https://github.com/renovatebot/renovate/assets/16151098/d604db4c-0c18-461e-8c5f-e1aca062bdb6">

Config:
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:best-practices",
    ":pinDependencies",
    ":automergeMinor",
    ":automergePr",
    ":rebaseStalePrs",
    ":semanticCommits",
    ":prHourlyLimitNone",
    ":enableVulnerabilityAlerts"
  ],
  "prConcurrentLimit": 5,
  "packageRules": [
    {
      "groupName": "kotlin",
      "matchPackagePrefixes": [
        "org.jetbrains.kotlin:"
      ]
    },
    {
      "groupName": "detekt",
      "matchPackagePrefixes": [
        "io.gitlab.arturbosch.detekt:"
      ]
    }
  ]
}
```

.github/CODEOWNERS
```
*       @mrclrchtr
```

It's odd because in the logs it says: "Skipping assignees and reviewers as automerge=true". I think it is a conflict between "assignAutomerge" and "assigneesFromCodeOwners".

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: Branch status yellow (branch="renovate/node-20.x")
DEBUG: Skipping assignees and reviewers as automerge=true (branch="renovate/node-20.x")
DEBUG: setPrCache() (branch="renovate/node-20.x")
```

</details>

### Have you created a minimal reproduction repository?

I have explained in the description why a minimal reproduction is impossible

**Comment by viceice** [comment, marked as answer]
this is an automatic thing done by GitHub, we can't change that.

**Reply by mrclrchtr** [reply]
Ah okay... I thought about this and searched fore some documentation about it, but could not find something..

Thx for the answer!
